eng/native/configurecompiler.cmake

```diff
@@ -525,7 +525,7 @@ if (MSVC)
 
   # disable C++ RTTI
   # /GR is added by default by CMake, so remove it manually.
-  string(REPLACE "/GR" "" CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS}")
+  string(REPLACE "/GR " " " CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS}")
```

Comment:
Intentional? Is this change needed?

Reply:
Yup, it is ported from old src/libraries/Native/Windows/CMakeFiles.txt. Without it, it replaces `/GR-` (set by default in VS2022) with `-` and that gives (non-fatal) warnings.